bigchaindb_driver/driver.py

```diff
@@ -193,6 +193,31 @@ def fulfill(transaction, private_keys):
         """
         return fulfill_transaction(transaction, private_keys=private_keys)
 
+    def get(self, *, asset_id, operation=None, headers=None):
+        """Get a list of transactions that have the given asset id and or
+        operation.
```

Comment:
> and or

Not exactly true, since the asset id is mandatory. I think I'd reword this to be something like:

> Get a list of transactions that are related to the given asset id (and optionally filtered by their operation).